### DISCUSSION TITLE
Multilingual MTEB Naming

### DISCUSSION THREAD

**Discussion by Muennighoff** [embeddings-benchmark/mteb#277] [General]
@apsdehal suggested that MMTEB may sound a bit like multimodal and XMTEB or XTEB may be better in case you have thoughts @KennethEnevoldsen @imenelydiaker ; We can also drop the acronym and just call it `Multilingual MTEB` or something :)
Perso I think the current one or Multilingual MTEB is fine but don't have too strong opinions.

**Comment by MartinBernstorff** [comment]
Phonetically, I'd prefer `XTEB`, but not a strong preference 👍

**Comment by KennethEnevoldsen** [comment]
Often I find that the "m" is typically used for multilingual (e.g. mbert), while "x" is used for cross-lingual (which is also partly true). It is best to keep it close to MTEB,  but xteb and mmteb are both fine for me.

**Reply by imenelydiaker** [reply]
It's true that historically in NLP "m" meant multilingual and "x" cross-lingual. It's just that we the recent advancements in LLMs the "M" is now used for Multimodal (MLLM for Multimodal LLM).

Maybe we should go with `Multilingual MTEB` to avoid all confusions.

**Comment by imenelydiaker** [comment]
I'd go for `Multilingual MTEB` to avoid all confusions.

**Reply by KennethEnevoldsen** [reply]
We can def. to that, people would short it down to MMTEB anyway.

Here is a somewhat decent cgpt suggestions:
> MUTEB (pronounced "mute-eb"): MUltilingual Text Embedding Benchmark. By rearranging the letters slightly, this version keeps the acronym pronounceable and similar to MTEB, with a clear emphasis on "Multilingual."

>GLOTEB: GLObal Text Embedding Benchmark. This version emphasizes the global, and thus multilingual, aspect of the benchmark. Pronounced "glow-teb."

I think GLOTEB is pretty good
